route/superadmin.js

```diff
@@ -15,65 +15,65 @@ router.use(roleUser.gte(konst.roleUser.superadmin))
 
 resources.forEach(resource => {
   router.get(`/${resource}/many`,
-    validate.query({
-      ids: joi.array().items(joi.number().integer()),
-    }),
-    async function (ctx) {
-      const { ids } = ctx.v.query
-      ctx.state.r = await adminRepo[resource].getMany(ids)
-    }
+  validate.query({
+    ids: joi.array().items(joi.number().integer()),
+  }),
+  async function (ctx) {
+    const { ids } = ctx.v.query
+    ctx.state.r = await adminRepo[resource].getMany(ids)
+  }
```

Comment:
I'm of idea that having function call and it's arguments with no indentation as above is (objectively) less readable. Having `CallExpression.arguments` to the default 1 would make much more sense to me.